### ISSUE TITLE
Any Remaining Work?

### ISSUE THREAD

**Issue by WildJimmy** [ubsuny/23-Homework7G4#63]
I am trying to make my plots render better in the jupyter notebook, but other than that and merging the last pull requests is there anything else to be done?

**Comment by WildJimmy** [comment]
Yep, it is all self contained in the jupyter notebook I have waiting on the most recent pull request https://github.com/ubsuny/23-Homework7G4/pull/67

**Comment by reshnashrestha** [comment]
It has been merged.
